packages/nx/src/command-line/release/config/version-plans.ts

```diff
@@ -54,7 +54,9 @@ const versionPlansDirectory = join('.nx', 'version-plans');
 
 export async function readRawVersionPlans(): Promise<RawVersionPlan[]> {
   const versionPlansPath = getVersionPlansAbsolutePath();
-  const versionPlansPathExists = await pathExists(versionPlansPath);
+  const versionPlansPathExists = await access(versionPlansPath)
+    .then(() => true)
+    .catch(() => false);
   if (!versionPlansPathExists) {
```

Comment:
Honestly can we just simplify this such that the if just checks `existsSync()`? I'm not sure why pathExists was used for this one in the first place, and we don't need to reuse `versionPlansPathExists`